clients/cloud/python/README.md

```diff
@@ -197,3 +197,34 @@ Consumed record with key alice and value 9,                       and updated to
     $ curl -u {{ SR_API_KEY }}:{{ SR_API_SECRET }} https://{{ SR_ENDPOINT }}/subjects/test2-value/versions/1
     {"subject":"test2-value","version":1,"id":100001,"schema":"{\"name\":\"io.confluent.examples.clients.cloud.DataRecordAvro\",\"type\":\"record\",\"fields\":[{\"name\":\"count\",\"type\":\"long\"}]}"}
     ```
+
+# Example 3: Run All the Above in Docker
+
+You also may run all the above code from within Docker.
+
+1. Copy to the current directory your local file with configuration parameters to connect to your Confluent Cloud instance.
+
+```bash
+$ cp $HOME/.confluent/librdkafka.config .
```

Comment:
You could consider instructing the user to just mount the config file directly from their home directory below instead of adding this copy step. 

docker run -v $HOME/.confluent/librdkafka.config:/root/.confluent/librdkafka.config -it --rm cloud-demo-python

Reply:
@rspurgeon *much* better yes.